contrib/cirrus/lib.sh

```diff
@@ -269,7 +269,7 @@ remove_packaged_podman_files() {
     req_env_vars OS_RELEASE_ID
 
     # If any binaries are resident they could cause unexpected pollution
-    for unit in io.podman.service io.podman.socket
+    for unit in podman.socket podman-auto-update.timer
```

Comment:
Are you sure the `io.podman.service` and `io.podman.socket` names aren't used in any CI distro.?  IIUC this function should gracefully handle invalid names for a distro. even if they're valid for another.

Reply:
Historical note: It's entirely possible these names came from Ubuntu, a very, very, very log time ago.

Reply:
We test fedora and debian, neither of those uses them. And no distro will use them because they all use upstream defaults (contrib/systemd/system), they would explicitly rename untis. These are legacy unit names for the old varlink socket.

Reply:
Ooooohhhh damn, the bloody varlink bites again! Lol, thanks for the clarification Paul :+1: